module/VuFind/src/VuFind/ILS/Driver/AbstractBase.php

```diff
@@ -81,4 +81,14 @@ protected function throwAsIlsException(
     ): void {
         throw new ILSException($msg ?? $exception->getMessage(), 0, $exception);
     }
+
+    /**
+     * Whether the driver supports the GetThis feature
+     *
+     * @return bool
+     */
+    public function supportsGetThis(): bool
```

Comment:
Do we need a separate method for controlling the GetThis feature at the ILS driver level, or can we base this on support for existing driver capabilities? If we're not adding any new forms of data retrieval to the driver, it doesn't feel like we need a new check method. If you're thinking of making this more configurable, we could probably achieve that at a higher level without having to impact all the ILS drivers.